weblate/trans/models/component.py

```diff
@@ -672,6 +677,7 @@ class Component(models.Model, PathMixin, CacheKeyMixin, ComponentCategoryMixin):
             "please consult the documentation for more details."
         ),
         validators=[validate_render_addon],
+        blank=True,
```

Comment:
We really don't want these to be blank at model level. If these need to be optional in a particular form, let's override it there, but not in the model as that has an impact on all users (for example API).